src/main/java/ch/uzh/ifi/hase/soprafs21/entity/User.java

```diff
@@ -28,8 +29,11 @@
 @Entity
 @NamedEntityGraph(name = "users_and_dogs_without_picture",
         includeAllAttributes = true,
-        attributeNodes = @NamedAttributeNode(value = "listOfDogs", subgraph = "dogs_without_picture_and_owner")
+        attributeNodes = {@NamedAttributeNode(value = "listOfDogs", subgraph = "dogs_without_picture_and_owner"),
+                @NamedAttributeNode(value = "tags", subgraph = "tags_without_owner")
+        }
```

Comment:
I think subgraph = "tags_wiout_owner" does not exist anymore

Reply:
removed in b2c9db3